app/View/Components/Photo.php

```diff
@@ -65,6 +65,7 @@ public function __construct(array $data)
 		$dim2x = '';
 		$thumb2x = '';
 
+		// TODO: The class Photo for the database model does not anymore contain the attributes `small`, `small_dim`, etc. Probably this code needs some fix/refactoring, too. However, where is this method invoked and what is the structure of the passed `data` array? (Could find any invocation.)
```

Comment:
This part is Work in Progress. It is part of the Livewire side. no worries I'll fix it.

Reply:
👍 Thx